rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -437,12 +606,92 @@ namespace rocal{
         m.def("GetIntValue",&rocalGetIntValue);
         m.def("GetFloatValue",&rocalGetFloatValue);
         // rocal_api_data_transfer.h
-        m.def("rocalCopyToOutput",&wrapper_copy_to_output);
-        m.def("rocalToTensor",&wrapper_tensor);
-        m.def("rocalToTensor32",&wrapper_tensor32);
-        m.def("rocalToTensor16",&wrapper_tensor16);
-        m.def("rocalCupyToTensor32",&wrapper_copy_cupy_tensor32);
-        m.def("rocalCupyToTensor16",&wrapper_copy_cupy_tensor16);
+        m.def("getOutputTensors", [](RocalContext context) {
+            rocalTensorList * tl = rocalGetOutputTensors(context);
+            py::list list;
+            unsigned int size_of_tensor_list = tl->size();
+            for (uint i =0; i< size_of_tensor_list; i++)
+                list.append(tl->at(i));
+            return list; 
+        });
+        m.def("getBoundingBoxCount",&rocalGetBoundingBoxCount);
+        m.def("getImageLabels", [](RocalContext context) {
+            rocalTensorList *labels = rocalGetImageLabels(context);
+            return py::array(py::buffer_info(
+                            (int *)(labels->at(0)->buffer()),
+                            sizeof(int),
+                            py::format_descriptor<int>::format(),
+                            1,
+                            {labels->size()},
+                            {sizeof(int) }));
+        });
```

Comment:
No return value policy required for boxes, labels API?

Reply:
In the case of returning a NumPy array from C++, if we don't specify a return value policy, the default return_value_policy::reference will be applied.